src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,280 @@
+  pnpm upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # updates the astro dependency:
+  yarn upgrade astro
+  # or, to update all dependencies:
+  yarn upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+This guide includes major highlights and instructions on how to handle breaking changes when upgrading from v1.x and v2.0-beta releases. See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.
+
+
+## Astro 2.0 Updates
+
+### Updated: Node Version >=16.12.0
+
+The minimum supported version of NodeJS is now 16.12.0.
+
+
+### Updated: Vite 4
+
+Astro v2.0 has upgraded from Vite 3 to [Vite 4](https://vitejs.dev/). We've handled most of the upgrade for you inside of Astro; however, some subtle Vite behaviors may still change between versions. 
+
+Refer to the official [Vite Migration Guide](https://vitejs.dev/guide/migration.html) if you run into trouble.
+
+
+## Astro 2.0 Breaking Changes
+
+### Removed: `legacy.astroFlavoredMarkdown`
+
+In 1.0 Astro moved the old Astro Flavored Markdown (also known as Components in Markdown) to a legacy feature. This change removes the `legacy.astroFlavoredMarkdown` option completely.
+
+In 2.0 this feature is not available in Astro at all. We recommend [migrating to MDX with the integration](/en/guides/integrations-guide/mdx/) for those were still using this feature in 1.x. See our older migration guide for more about [converting existing Mardown files to MDX](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx#converting-existing-md-files-to-mdx).
+
+### Removed: `Astro.resolve()`
+
+In 0.24 Astro deprecated `Astro.resolve()`. This change removes the option completely.
+
+In 2.0 this feature is not available in Astro at all. You can resolve asset paths using `import` instead. For example:
+
+```astro
+---
+import 'style.css';
+import imageUrl from './image.png';
+---
+
+<img src={imageUrl} />
+```
+
+See the [v0.24 migration guide](https://github.com/withastro/docs/blob/aa7e4f568f418657eba297778729a8aee2a36409/src/pages/en/migrate.mdx#deprecated-astroresolve) for more information.
+
+### Removed: `Astro.fetchContent()`
+
+In 0.26 Astro deprecated `Astro.fetchContent()`. This change removes the option entirely.
+
+In 2.0 this feature is not available in Astro at all. Use [`Astro.glob()`](/en/guides/imports/#astroglob) instead to fetch Markdown files, or migrate to the [Content Collections](/en/guides/content-collections/) feature.
+
+```js
+let allPosts = await Astro.glob('./posts/*.md');
+```
+
+### Removed: `Astro.canonicalURL`
+
+In 1.0 Astro deprecated `Astro.canonicalURL`. This change removes the option entirely.
+
+In 2.0 this feature is not available in Astro at all. Use `Astro.url` instead to construct the canonical URL.
+
+```js
+const canonicalURL = new URL(Astro.url.pathname, Astro.site);
+```
+
+### Added: `_astro/` folder for build assets
```

Comment:
Better called `Changed`, since this was technically renamed?

Reply:
Maybe! Although the problem was there was no, single folder for these before. They ended up in various places. I kind of think "added" makes sense here. Anyone else have thoughts?